applications/StructuralMechanicsApplication/python_scripts/adaptative_remeshing_structural_mechanics_static_solver.py

```diff
@@ -3,21 +3,11 @@
 # Importing the Kratos Library
 import KratosMultiphysics
```

Comment:
Should this really be a solver ? 
Cannot it be done via a process ?

Reply:
Nope, it replaces functions from the base solver